src/backend/access/transam/parallel.c

```diff
@@ -101,6 +101,9 @@ typedef struct FixedParallelState
 
 	/* Maximum XactLastRecEnd of any worker. */
 	XLogRecPtr	last_xlog_end;
+
+	/* Track if parallel worker is spawned in the context of Babelfish */
+	bool babelfish_context;
```

Comment:
if this is not in the context of mutex, move it before mutex

Reply:
Thats good point, updated accordingly. Thanks